getting_started/setup/setup_up_2_board.rst

```diff
@@ -28,10 +28,16 @@ will show UP-APL01 R3.6.
    
      this link needs to be updated to something accessible externally
 
-* Press F2 to enter BIOS main menu and check the BIOS version.
-* Press ENTER when prompted for password.
-* If the current BIOS version is older than 1.8, please update to 1.8
+* If the current BIOS version is older than 1.8, please update to v1.8
   before flashing v3.6.
+  .. note::
```

Comment:
Hi @stripes416,

I don't think the word "either" is necessary and the sentence "check your BIOS version press" needs a ":" before the list starts. 

There are a couple of issues here that impact rendering:

1. There needs to be a carriage return between v3.6. and .. note::
2. Bullets seem unnecessary in note section.
3. Make sure that the indentations render correctly in the final build.
4. Is the "Press ENTER" portion supposed to be in the note?